content/docs/command-reference/exp/gc.md

```diff
@@ -0,0 +1,59 @@
+# exp gc
+
+Remove `dvc experiments` from the <abbr>project</abbr>.
+
+## Synopsis
+
+```usage
+usage: dvc exp gc [-h] [-q | -v] [-w]
+                  [-a] [-T] [--all-commits] [--queued] [-f]
```

Comment:
Why no `--cloud` (or `--git-remote`) option? Since there's `exp push/pull`.

How can you delete experiments from a remote repo? I guess you have to clone it, `exp gc` locally, and `exp push` syncs it up in that case, removing remote exps?

Reply:
There is no way to remove pushed remote experiments at the moment (other than using `git`).

There was some discussion over whether the ability to remove them belongs in `exp gc/remove` (like removing DVC data objects from a DVC remote), or in `exp push` (like removing branches or tags from a git remote).

In git, to remove a remote branch or tag, you have to use `git push <remote_name> --delete <ref_name>`, so for removing DVC experiments from a git remote, it should probably work the same way.

Reply:
> whether the ability to remove them belongs in exp gc/remove or in exp push

I vote for `exp gc -c` for consistency with plain `gc`. Maybe the commands could even get merged at some point.